ublox/src/ubx_packets/packets.rs

```diff
@@ -402,6 +402,137 @@ enum NavStatusFlags2 {
     Inactive = 3,
 }
 
+/// Information message conifg
+#[ubx_packet_recv_send]
+#[ubx(
+    class = 0x06,
+    id = 0x2,
+    fixed_payload_len = 10,
+    flags = "default_for_builder"
+)]
+struct CfgInf{
+    protocol_id: u8,
+    reserved: [u8; 3],
+    #[ubx(map_type = CfgInfMask)]
+    inf_msg_mask_0: u8,
+    #[ubx(map_type = CfgInfMask)]
+    inf_msg_mask_1: u8,
+    #[ubx(map_type = CfgInfMask)]
+    inf_msg_mask_2: u8,
+    #[ubx(map_type = CfgInfMask)]
+    inf_msg_mask_3: u8,
+    #[ubx(map_type = CfgInfMask)]
+    inf_msg_mask_4: u8,
+    #[ubx(map_type = CfgInfMask)]
+    inf_msg_mask_5: u8,
+}
+
+#[ubx_extend_bitflags]
+#[ubx(from, into_raw, rest_reserved)]
+bitflags! {
+    /// `CfgInfMask` parameters bitmask
+    #[derive(Default)]
+    pub struct CfgInfMask: u8 {
+        const ERROR= 0x1;
+        const WARNING = 0x2;
+        const NOTICE  = 0x4;
+        const TEST  = 0x8;
+        const DEBUG  = 0x10;
```

Comment:
Looking at the u-blox 7 protocol spec, it looks like these two might be swapped?
![image](https://user-images.githubusercontent.com/879600/131769591-d9319049-51b8-4acd-a4db-a0e9c9d0aa90.png)

Reply:
Good catch. Fixed.